index.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: black;
```

Comment:
- [ ] Good work 😃 but here is one thing left that needs to be done. You can use this opportunity to change it to inline CSS.